PrReviewCode/WeatherForecast.cs

```diff
@@ -8,5 +8,5 @@ public class WeatherForecast
 
     public int TemperatureF => 32 + (int)(TemperatureC / 0.5556);
 
-    public string? Summary { get; set; }
-}
\ No newline at end of file
+    public string? Summary { get; set; } = throw new Exception("Hello world");
```

Comment:
Throwing an exception in a property getter is an unusual and likely unintended pattern that will prevent object instantiation.

 **SUGGESTION:**

```suggestion
 public string? Summary { get; set; }
```